### ISSUE TITLE
existing EMCO settings overwrite v1.6.0 updates

### ISSUE THREAD

**Issue by iLPdev** [iLPdev/prs#93]
this happens on uninstall/install of updated package from 1.5.0 to 1.6.0. is a quirk of EMCO.

**Comment by iLPdev** [comment]
```
local latestRevision = 3 -- this is the part you increment when you want to apply new changes
local revisionFile = io.open(getMudletHomeDir() .. "/prsrevision.txt", "r") -- open file in read mode
local currentRevision = revisionFile:read("*a") -- read the contents
revisionFile:close() -- close the file
currentRevision = tonumber(currentRevision:trim()) or 0 -- make it a number, or assign it 0. This is better handled by checking for file existence and such in the above code but this is meant to be pseudocode
if currentRevision < latestRevision then
  if currentRevision < 1 then
    -- all the changes you made to how it works in revision 1 go here
  end
  if currentRevision < 2 then
    -- all the changes you made to how it works in revision 2 go here
  end
  if currentRevision < 3 then
    -- all the changes you made to how it works in revision 3 go here
  end
  revisionFile = io.open(getMudletHomeDir() .. "/prsrevision.txt", "w+")
  revisionFile:write(latestRevision)
  revisionFile:close()
end
```

call the :load, then check revision, and then call save at the end of all the revisions. Like after writing out the latestRevision

**Comment by iLPdev** [comment]
closed by #101
